src/DotNetCore_Template.Tests/TemplateTest.cs

```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace DotNetCore_Template.Tests
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void Test1()
+        {
+            Assert.Pass();
+        }
+    }
+}
```

Comment:
Use newline at the end of a file

```suggestion
}

```